src/fresh.js

```diff
@@ -423,7 +446,7 @@ class Fresh {
         const animation = await pin(stripIpfsUriPrefix(animationURI));
         spinner.succeed(`📌 ${animation} was pinned!`);
       }
-
+      this.datastore.update({ tokenId }, { pinned: true });
```

Comment:
IPFS pinning is idempotent, but update here just in case we want to use this information later on.

Reply:
Good call. 👍